test/c/test_substrait_c_api.cpp

```diff
@@ -50,3 +54,39 @@ TEST_CASE("Test C Get and To Json-Substrait API", "[substrait-api]") {
 
   REQUIRE_THROWS(con.FromSubstraitJSON("this is not valid"));
 }
+
+TEST_CASE("Test C SelectFromValues Get and To Substrait API", "[substrait-api]") {
+  DuckDB db(nullptr);
+  SubstraitExtension substrait_extension;
+  substrait_extension.Load(db);
+  Connection con(db);
+  con.EnableQueryVerification();
+
+  auto json = con.GetSubstraitJSON("SELECT 13 as id, 17 as age");
+
+  auto result = con.FromSubstraitJSON(json);
+
+  REQUIRE(CHECK_COLUMN(result, 0, {13}));
+  REQUIRE(CHECK_COLUMN(result, 1, {17}));
+}
+
+TEST_CASE("Test C CTAS Get and To Substrait API", "[substrait-api]") {
+  DuckDB db(nullptr);
+  SubstraitExtension substrait_extension;
+  substrait_extension.Load(db);
+  Connection con(db);
+
+  auto proto = con.GetSubstrait("create table t1 as SELECT * FROM (VALUES ('john', 25), ('jane', 21))");
+  auto result = con.FromSubstrait(proto);
+
+  REQUIRE_NO_FAIL(con.Query("SELECT * from t1"));
+
+  auto json_plan = con.GetSubstraitJSON("create table t2 as SELECT * FROM t1");
+  result = con.FromSubstraitJSON(json_plan);
+
+  proto = con.GetSubstrait("select * from t2 limit 2");
+  result = con.FromSubstrait(proto);
+
+  REQUIRE(CHECK_COLUMN(result, 0, {"john", "jane"}));
+  REQUIRE(CHECK_COLUMN(result, 1, {25, 21}));
```

Comment:
let's add some additional tests similar to other items (python?)

Reply:
Added C tests. Working on python tests.
Raised a new [PR](https://github.com/scgkiran/duckdb-substrait/pull/2) stacked on top of [PR for cmake changes](https://github.com/sundeck-io/duckdb-substrait/pull/4)